products/rhel8/profiles/anssi_bp28_high.profile

```diff
@@ -23,15 +23,25 @@ selections:
     - anssi:all:high
     - var_password_hashing_algorithm=SHA512
     - var_password_pam_unix_rounds=65536
+
     # the following rule renders UEFI systems unbootable
     - '!sebool_secure_mode_insmod'
     - '!timer_logrotate_enabled'
+
     # disable R45: Enable AppArmor security profiles
     - '!apparmor_configured'
     - '!all_apparmor_profiles_enforced'
     - '!grub2_enable_apparmor'
     - '!package_apparmor_installed'
     - '!package_pam_apparmor_installed'
+
+    # An alternative solution for R67 is using nss-pam-ldapd package, in this case ensures SSL and certificate configuration
+    - ldap_client_start_tls
+    - ldap_client_tls_cacertpath
+
+    # Ensure nis is not used for RHEL 8 in R69
+    - no_nis_in_nsswitch
```

Comment:
I think this is not needed. as it was added in the "related rules" section in anssi control file

Reply:
Hi @Xeicker , previously it was included only for `rhel8` and `ol8` via Jinja2 in the `anssi` control file. So I removed the exception from the control file and included the rule directly in the product profiles. Do you think we should also include it for `ol8` profiles? If so, I can quickly update it.

Reply:
Hi @Xeicker , do you think that is also necessary to update ol8 profiles in this case?